Arduino/PulseInSync/PulseInSync.ino

```diff
@@ -7,17 +7,12 @@
 // Arduino Nano
 RF24 radio(7, 8);  // CE=7, CSN=8
 
-//Arduino Uno
-//RF24 radio(9, 8);  // CE, CSN
-
-
 // N total address for nodes to read/write on.
-// TODO: configure per radio using!
-int nodeNumber = 2; // TODO: switch between 0 and N depending on radio
-const int N = 6;  // Reading only supported on pipes 1-5
+int nodeNumber = 1; // Changes between 0 and N depending on radio
 // Addresses through which N modules communicate.
 // Note: Using other types for addressing did not work for reading from multiple pipes.  Why?  IDK but this works ;-)
-long long unsigned addresses[6] = {0xF0F0F0F0F0, 0xF0F0F0F0AA, 0xF0F0F0F0BB, 0xF0F0F0F0CC, 0xF0F0F0F0DD, 0xF0F0F0F0EE};
+long long unsigned addresses[6] = {0xF0F0F0F0F1, 0xF0F0F0F0AA, 0xF0F0F0F0BB, 0xF0F0F0F0CB, 0xF0F0F0F0DC, 0xF0F0F0F0ED};
```

Comment:
leave out this change for now but definitely experiment with the different addresses in the arduino test scripts!